example/python/permissions/can_add_asset_qty.py

```diff
@@ -0,0 +1,43 @@
+#
+# Copyright Soramitsu Co., Ltd. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+
+import iroha
+import commons
+
+admin = commons.user('admin@test')
+alice = commons.user('alice@test')
+
+
+def genesis_tx():
```

Comment:
That function is pretty similar among the files, that better be moved somewhere

Reply:
Finally, it was decided not to modify this place. Moving of the common part will lead to hiding command of domain creation that includes the test role. Otherwise, there will be nothing to move.

Thus, in order to increase examples' demonstrativeness, genesis functions would not be modified.